src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -76,7 +76,11 @@ exports.callCallbacks = function(queue : UpdateQueue, context : any) {
     if (callback && !node.callbackWasCalled) {
       node.callbackWasCalled = true;
       if (typeof context !== 'undefined') {
-        callback.call(context);
+        try {
+          callback.call(context);
+        } catch (error) {
+          // do something
```

Comment:
@gaearon `ReactFiberCommitWork.js` returns the error.

```js
 try {
      instance.componentDidUpdate(prevProps, prevState);
      return null;
    } catch (error) {
      return error;
    }
```

But I can't return the error as it'll stop the `while` loop of the subsequent `callbacks` of the rest of the nodes in the queue.

Should I make `node.callbackWasCalled = false` if it throws up and `console.log`? Suggestions please!